### ISSUE TITLE
Export segment to GPS button is not displaying correctly.

### ISSUE THREAD

**Issue by piaseq** [thomaschampagne/elevate#923]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: _____
  * Chrome/Opera/Chromium version: _____
  * OS version: _____
  
#### Steps to reproduce:

  1. Enter activity details
  2. Open any segment
  3. Check how Export to GPS button looks like
  
#### Actual Behavior:

  * What happened?
Export segment to GPS button is not displaying correctly.
  * Do you have javascript errors logs in browser console?

  ```javascript
Access to font at 'https://d3nn82uaxijpm6.cloudfront.net/assets/maison-neue/MaisonNeueWEB-BoldItalic-fc390f34f8d4a1d561e216afcffc49ae7594df79159b6193406df69e66a7994a.woff2' from origin 'https://www.strava.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
MaisonNeueWEB-BoldItalic-fc390f34f8d4a1d561e216afcffc49ae7594df79159b6193406df69e66a7994a.woff2:1 Failed to load resource: net::ERR_FAILED
  ```
  
#### Expected Behavior:

  * What did you expect to happen?
  
#### Media attachement:

![image](https://user-images.githubusercontent.com/11159401/75159648-d73aad80-5718-11ea-94b7-a195f3cc2511.png)

#### Providing your Elevate synced athlete activities can be useful to solve the problem

> To export your activities click on "sync button" (top right and corner), then "Backup activities". You can zip the file and upload it here.
